### PR TITLE
Added tests for canvasToPixel, pixelToCanvas, pageToPixel

### PR DESCRIPTION
**Pull request by posegae**
Added tests for the point conversion files in src.

Would be happy to make any changes necessary or add additional documentation, just let me know if that's needed.

### CODE REVIEW

File: test/pixelToCanvas_test.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      y: 30 });
+    assert.deepEqual(convertedPoint2, {
+      x: 0,
+      y: 0 });
+    assert.deepEqual(convertedPoint3, {
+      x: 0,
+      y: 128 });
+  });
+
+  it('should fail to convert points in the canvas coordinate system to non-corresponding points in the pixel coordinate system', function () {
```
Comment (swederik) [inline test/pixelToCanvas_test.js]: I'm not sure I understand the value of this specific test. It converts points to canvas coordinates, and then checks that they don't equal incorrect coordinates. Isn't the test above sufficient?

**Comment by swederik** [comment]
This is great! Thanks! I made one minor comment because I don't understand the value of the last test for pixelToCanvas.

**Comment by posegae** [comment]
Thanks for looking over it! The last tests for both pixelToCanvas and canvasToPixel were leftover from an earlier approach to the tests, but are definitely redundant now that the comparison point values are hardcoded in. I've updated the tests to reflect that.

**Comment by swederik** [comment]
Thanks!

**Pull request merged**